code/src/Components/Movie.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Movie = ({id, title, release_date, poster_path}) => {
+      return (
```

Comment:
It's a little bit hard to see here on GitHub, but maybe some more indentation is needed in this return? :)